superduperdb/vector_search/base.py

```diff
@@ -6,7 +6,9 @@
 
 import numpy
 import numpy.typing
-import torch
+
+if t.TYPE_CHECKING:
```

Comment:
Ooops, I intended to start a review.

Same issue here, again, I think we should just generalize the code to `t.Any`, and then always have an `else` in these chains if `isinstance` and `hasattr` where we raise a `TypeError`

Reply:
Same as my last comment.